src/index.js

```diff
@@ -152,6 +153,11 @@ function ParseServer({
   const hooksController = new HooksController(appId, collectionPrefix);
   const userController = new UserController(emailControllerAdapter, appId, { verifyUserEmails });
 
+  // Validate session length, default to 1 year if not set
+  sessionLength = Number(sessionLength)
+  if(isNaN(sessionLength) || sessionLength <= 0) {
+    sessionLength = 31536000000;
```

Comment:
you should add the validation that the sessionLength is valid in Config.validate, and throw an error if the session length is not valid